tmate/install.sh

```diff
@@ -14,6 +14,34 @@ sudo adduser --disabled-password --gecos "" $UNAME --shell /bin/bash
 ###Add villager to dialout group###
 sudo usermod -a -G dialout $UNAME
 
+su $UNAME -c cat <<"EOF" >> /home/$UNAME/.bashrc
```

Comment:
It's the appending part of the command that needs to be run as `$UNAME`. Use `cat <<"EOF" | sudo -u $UNAME tee -a ~/.bashrc` instead. Or `cat <<"EOF" | sudo -u $UNAME tee -a ~/.bashrc > /dev/null` if you prefer it to be silent.

Reply:
Committed with a minor change